com.stansassets.build/BuildPipeline/ViewModels/IBuildStepsViewModelProvider.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace StansAssets.Build.Pipeline
+{
+    public interface IBuildStepsViewModelProvider
```

Comment:
I do not like the idea of yet another `ISomeProvider` and `ISomeContainer`, especially when the data differs only in type, but I think we should firstly finish #22 and #23 in order to clarify the usages of providers and containers.